executable/vba.ksy

```diff
@@ -0,0 +1,1209 @@
+meta:
+  id: vba
+  title: Visual Basic exe structures
+  application: Visual Basic
+  file-extension: exe
+  xref:
+    wikidata: Q2378
+  endian: le
+  license: Apache-2.0
+  encoding: ascii
+  ks-opaque-types: true
+doc: |
+  Converted from https://github.com/williballenthin/python-vb/blob/master/vb/__init__.py with addition of some structures described in various reverse-engineered documentation
+  
+  # an external script converting raw virtual addresses to offsets in a file is needed, save it as resolve_offset.py
+  from kaitaistruct import KaitaiStruct, KaitaiStream
+  
+  class ResolveOffset(KaitaiStruct):
+    __slots__ = ("ofs",)
+    def __init__(self, rawAddress: int, root, _io: KaitaiStream, _parent=None, _root=None):
+      _root = root
+      self._parent = _parent
+      self.ofs = _root.pe.get_offset_from_rva(rawAddress - _root.pe.OPTIONAL_HEADER.ImageBase)
+
+doc-ref:
+  - https://github.com/williballenthin/python-vb/blob/master/vb/__init__.py
+  - https://web.archive.org/web/20071020232030if_/http://www.alex-ionescu.com/vb.pdf
+  - https://www.vb-decompiler.org/pcode_decompiling.htm
+  - https://sysenter-eip.github.io/VBParser#visual-basic-5-6-file-format-information
+params:
+  - id: pe
```

Comment:
Omitting `type` means byte array in Kaitai Struct (https://github.com/kaitai-io/kaitai_struct/issues/686), but I see you're doing things like `_root.pe.get_offset_from_rva()`. Even if you specified some opaque type, this again won't work from the same reason.

Reply:
Thanks.